assets/js/theme/common/product-details-base.js

```diff
@@ -373,7 +374,10 @@ export default class ProductDetailsBase {
     }
 
     updateWalletButtonsView(data) {
-        this.toggleWalletButtonsVisibility(data.purchasable && data.instock);
+        const viewModel = this.getViewModel(this.$scope);
+        const isValidForm = viewModel.$addToCartForm[0].checkValidity();
```

Comment:
more like a small suggestion here. Perhaps the use of a method like `.first()` would be better, wdyt? 💅
`const isValidForm = viewModel.$addToCartForm.first().checkValidity();`